src/dotnet-roslyn-tools/Commands/PRTaggerCommand.cs

```diff
@@ -2,29 +2,27 @@
 // The.NET Foundation licenses this file to you under the MIT license.
 // See the License.txt file in the project root for more information.
 
-namespace Microsoft.RoslynTools.Commands;
-
-using System.CommandLine.Invocation;
 using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.Net.Http.Headers;
 using Microsoft.Extensions.Logging;
-using Microsoft.RoslynTools.Products;
+using Microsoft.RoslynTools.Utilities;
+
+namespace Microsoft.RoslynTools.Commands;
 
 internal static class PRTaggerCommand
 {
     private static readonly PRTaggerCommandDefaultHandler s_prTaggerCommandHandler = new();
-
-    private static readonly Option<string> VSBuild = new(new[] { "--build", "-b" }, "VS build number") { IsRequired = true };
-    private static readonly Option<string> CommitId = new(new[] { "--commit", "-c" }, "VS build commit") { IsRequired = true };
+    public static readonly Option<int> maxVsBuildCheckNumber = new(new[] { "--vsBuildCheckNumber" }, () => 20, "Maximum number of VS build to check. Tagger would compare each VS build and its parent commit to find the inserted payload.");
 
     public static Symbol GetCommand()
     {
         var command = new Command("pr-tagger", "Tags PRs inserted in a given VS build.")
         {
-            VSBuild,
-            CommitId,
             CommonOptions.GitHubTokenOption,
             CommonOptions.DevDivAzDOTokenOption,
-            CommonOptions.DncEngAzDOTokenOption
+            CommonOptions.DncEngAzDOTokenOption,
+            maxVsBuildCheckNumber
```

Comment:
I believe this ordering influences how the help (-?) text is generated. You may want the command specific option to show before the general options.